assets/src/edit-story/components/panels/layer/layerList.js

```diff
@@ -59,7 +61,7 @@ function LayerPanel() {
       const message = sprintf(REORDER_MESSAGE, position);
       speak(message, 'assertive');
```

Comment:
^ I suppose `assertive` is not necessary here anymore, just `speak(message)` should do?

Later would potentially be good to be able to just import `speak` directly without having to do `const { speak } = useLiveRegion('assertive');`, however, currently it seems to be used in one place only.

Reply:
Ah, good catch 👍 

Importing like that didn't work well with `renderHook` in tests for some reason. But I'll check again.

Reply:
Fixed now 🎉